osu.Game/Skinning/SkinManager.cs

```diff
@@ -144,14 +144,7 @@ private void populateMetadata(SkinInfo item)
 
         protected override void PreImport(SkinInfo model)
         {
-            var dbContext = ContextFactory.Get();
-
-            // Workaround System.InvalidOperationException
-            // The instance of entity type 'FileInfo' cannot be tracked because another instance with the same key value for {'ID'} is already being tracked.
-            foreach (var file in model.Files)
-            {
-                file.FileInfo = dbContext.FileInfo.Find(file.FileInfoID);
-            }
+            model.Files.Requery(ContextFactory);
```

Comment:
does this have to be on `Files` specifically? or can `model.Requery` be called instead? if so, would feel safer to me.